pkg/spdk/backup.go

```diff
@@ -0,0 +1,422 @@
+package spdk
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strconv"
+	"sync"
+
+	bitmap "github.com/hashicorp/nomad/nomad/structs"
```

Comment:
Can we use other bitmap lib instead? how about https://pkg.go.dev/github.com/kelindar/bitmap

Reply:
Replaced. Thank you.